server/Cargo.toml

```diff
@@ -2,5 +2,13 @@
 name = "server"
 version = "0.1.0"
 authors = ["Russell Johnston <rpjohnst@gmail.com>"]
+build = "build.rs"
+
+[build-dependencies]
+serde_codegen = "*"
```

Comment:
Should set a minor version for all dependencies (`"0.8"` for `serde_codegen` for example).